README.md

```diff
@@ -1,6 +1,61 @@
+
+The system understands the role of the person both administering the test, and registering the result, allowing for different confidence levels to establish:
+* Did this person really take the test, or did the test sample come from someone else?
+* Was the result observed and entered by a trusted professional?
+
+#### In person tests
+* The patient registers their details in the application
+* The medical professional scans the patients profile
+* The medical professional administers the test and adds the resuls
+* The results appear on the patients device
+
+#### Tests via post
+* The test administrator generates a number of unique QR codes for their tests, which are printed out with test instructions.
+* One printed QR code is sent out with each test.
+* The patient recives the test in the post.
+* The patient registers in the application (they can be already registered).
+* The patient scans the printed QR code to register the test.
+* The patient administers the test, following the instructions.
+* Depending on whether this test provides immediate results, they either:
+  * Submit their results through the application.
+  * Post back their test sample with the original QR code.
+    * On reciept of the test sample, a medical professional processes the sample.
+    * The medical professional scans the printed QR code, and then enters the rest results.
+    * The test results appear in the patients application.
+    
+![image](https://user-images.githubusercontent.com/1318111/78276614-2e4b5380-750b-11ea-98f8-3e258bdfb1f6.png)
+
+### Test results ###
+Users can see a list of their test results indicating whether antigens or antibodies were found, depending on the test type. If a medical professional added notes these are also available to view.  Along with links to additional information on understanding test results.
 
-## Available Scripts
+![image](https://user-images.githubusercontent.com/1318111/78277018-bcbfd500-750b-11ea-9e0c-54f08f9d2092.png)
+
+### Immunity passport ###
+Access to test results can be shared with others.  On the users profile page there is a short-lived QR code that grants temporary access to view & confirm another user's status.  The second user (the checker) takes a photo of this QR code, granting them access to view the test results of the first user (the patient).
+
+At this point, if the 'checker' is a medical professional with sufficient priviledges, they can also add new verified test results for the patient (facillitating the in-person test flow).
+
+![image](https://user-images.githubusercontent.com/1318111/78276524-107dee80-750b-11ea-88a2-cdb056428221.png)
+
+
+## Team ##
+
+This application was designed and built by a team of engineers from [transferwise.com/](https://transferwise.com/) in partnership with a team of medical professionals.
```

Comment:
This application was designed and built by a team of engineers from [transferwise.com/](https://transferwise.com/) in partnership with a team of medical professionals. To learn more about the project and the team, check out [cov-clear.com](https://cov-clear.com/about/)